packages/google-cloud-bigquery-datapolicies/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@google-cloud/datapolicies",
```

Comment:
Is this library name correct? Or should it be `bigquery-datapolicies`? Or `bigquery-data-policies`?

Reply:
Or `bigquery-data-policy` (the sample prefix is `bigquerydatapolicy`)